resources/components/App.vue

```diff
@@ -79,13 +83,19 @@ module.exports = {
 		'card-stack': CardStack
 	},
 
-	computed: $.extend( {}, mapState( [
+	computed: $.extend( {
+		showTabs: function () {
+			return this.user.isAuthenticated && this.user.isAutoconfirmed;
+		},
+		loginMessage: function () {
+			return mw.config.get( 'wgMVSuggestedTagsLoginMessage' );
```

Comment:
This login message is going to require some ugly workarounds unless we can change it to get rid of the wikitext. Apparently the Readers Web team prefers to pass in URLs as parameters instead of building them in place with wikitext for similar reasons.

Until we can change how the message is generated, I think that this approach is fine. Treating `loginMessage` as a computed property seems more appropriate than treating it as `data` here. Using `v-html` in the template above is necessary as well. I'd recommend just adding a single-line linter override on line 33.

One other thing about calling `mw.config.get` directly in a Vue component: now that component is dependent on mw.config, so we'll need to ensure that it's stubbed out for testing. I *think* that `setup.jest.js` does this already but if not we'll need to add a stub that returns an empty string here. I generally prefer to make these kinds of dependencies explicit with plugins rather than relying on globals, but it's not worth doing for just one edge case.

I guess the real take-away here is that we should avoid wikitext in messages wherever possible!